### ISSUE TITLE
scons: SHA256 mismatch

### ISSUE THREAD

**Issue by xfantasy** [Homebrew/homebrew#41772]
OS X 10.11

#: brew install --force scons
_________________________________________________________________
Warning: You are using OS X 10.11.
We do not provide support for this pre-release version.
You may encounter build failures or other breakage.
==> Downloading https://downloads.sourceforge.net/scons/scons-2.3.5.tar.gz
Already downloaded: /Library/Caches/Homebrew/scons-2.3.5.tar.gz
<strong>Error: SHA256 mismatch</strong>
Expected: 5b72f959cafbef81f408b503bc8e8d5cfc39f41fb5b629e9ff13bdf20a3eefe2
Actual: 8a8993e1914801ace5ce83c92bf4c43127669750e9dec8eb93574e57729e9c42
Archive: /Library/Caches/Homebrew/scons-2.3.5.tar.gz
To retry an incomplete download, remove the file above.
_________________________________________________________________

**Comment by bfontaine** [comment]
I can’t reproduce.

```
$ brew fetch -s scons
==> Downloading https://downloads.sourceforge.net/scons/scons-2.3.5.tar.gz
==> Downloading from http://netcologne.dl.sourceforge.net/project/scons/scons/2.3.5/scons-2.3.5.tar.
######################################################################## 100,0%
Downloaded to: /Library/Caches/Homebrew/scons-2.3.5.tar.gz
SHA1: 12fc5b506d895795dd6315f11f8fc3c4dd453c78
SHA256: 8a8993e1914801ace5ce83c92bf4c43127669750e9dec8eb93574e57729e9c42

$ curl -sL https://downloads.sourceforge.net/scons/scons-2.3.5.tar.gz | shasum -a 256
8a8993e1914801ace5ce83c92bf4c43127669750e9dec8eb93574e57729e9c42  -
```

**Comment by dunn** [comment]
This was fixed in fada9441f00215430e5f76dbb68a850df036ef1c; a quick `brew update` should fix it!
